beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -1,6 +1,13 @@
 import { useTheme } from "@mui/material";
 import * as React from "react";
+import countryMap from "../../../data/map/variants/classic/CountryMap";
```

Comment:
Responded on the last one: "VSCode automatically imports files like that, in this case using an .env might be more work as the paths would have to be written out manually."

I'm open to suggestions on how this can be improved though, but I'm not sure how an .env file would help since files are all at different directory levels.

Reply:
It seems there are ways to do this either through webpack or by ejecting the react app, but that is out of the scope of this PR. We should make note of this as tech debt though, I agree it would be a lot cleaner if we didn't have those relative paths in every import.